static/docs/commands-reference/add.md

```diff
@@ -119,6 +119,7 @@ $ tree
 
 Let's check the `data.xml.dvc` file inside:
 
+<!-- prettier-ignore -->
```

Comment:
why do we need to ignore this?

Reply:
Otherwise the YAML blocks (DVC-file samples) get indented by prettier. See the 2nd point in the description of #363

Reply:
I'm not sure this is a problem - looks like this is a standard prettier indentation for YAML: 

https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBadAdKBLKA3CbMOAAjKTJIGYAWADhqqwBMBDeSiygJgAZeAjKkGpuTKACNsAG2moYELmQBk2ZvwFZKJAObY8CbRQDCACwBO2AM5bKAM1YBbGQE9yJACIBXR63M2obVZmZnM4KyskW20yaVxwigAfaJjtGgBWOhIAdVZpAGtfQI9zADoU1LIAZS9sDgBibgBObgrKMDq3SooAJQgXPJIAeVZ8trIrGHZSGIoAWQBJcZIABwhJwZIKel4aADYsK1NsFflFEgAqNQ0sFfMIZi8wGCjAylQSFLarfK9K9wAQgAZKhNGgeNoAfhIAEcvKxYJ1tG1tm1mOEwJYVjBsNAtiQAawfnAYBI8tIoatLMRKiiSBleKV+G0WW8yB9vr9-u4yMCaIw6AAJNpwhE4mBdIxkTRsyjoqyYk446BKSg1FZwcwkQUQCArNp3IgzbpkMQtJm8VmpLBTAAePO46QEpRorVgECm0nx-MYpXSbsgjkcCBgkRIAD4UkDpiRWHZ4OYoLrAtYSBJwjByrLCWBfisSJBYKxniRUwA5BFgFwpAEyaRWYNagAC1CodFQjDbWZAABoQHrlVArMhQH57gB3AAKfgQw5QrAIal7IAk5mL+RJVRWxdwOmQMHMXjgfdMMEc0myx3gVm3xCqs7q+k6yHAEWXuCsmpgk7XOl8yAcetjxAAArKxbQBNdc03Jw4CBeIALyT8+zA20ql3aQ4AARS8D04EQoC+23fxNRfAZz2XQ1YGyNQYFMZA6F4Ij7k-bI1xWF87nCTUDGXOE8J-PU5xAIlUCgOA4HRZhlzCOFsDCH9WD-VgCOQkBP2cfdD2AqwMOw3D4FU4CpgkGjmDo5BuD7A9WBkXdjAgIMVJQJNxOXLxPwAFVYCQ50Az8AF8AqAA

Reply:
Yes its standard prettier indentation and can't be configured otherwise, only ignored.

I thought we wouldn't want the YAML DVC-file blocks to be changed by prettier since they are just textual copy-pastes of how the real files look. prettier changes them in a way that DVC does never produce them.

BTW we may have similar problems with other code blocks, I just ran `npm run prettier-md` and I'm examining each change... Will report

Reply:
I think it's fine. Don't has to be exactly the same including indents. It's easier to let it format this - so that we always have the same standard representation everywhere.

Reply:
OK will un-do then 👍 

p.s. the rest of the changes prettier does are fine, but I had to manually fix https://github.com/jorgeorpinel/dvc.org/blob/master/static/docs/changelog/0.35.md which is all messed up apparently 😬

Reply:
Un-done. And prettier-md script ran on everything.